### PR TITLE
Don't use double semicolon

### PR DESCRIPTION
**Pull request by anchpop**
## Depends

- [x] #476

## Problem

`reason_to_ocaml.sh` generates `;;` everywhere. But I think that's kind of unidiomatic - it would be better to use `let () = ` in most cases. 

## Solution

Convert every usage of `;;` to `let () = `

### CODE REVIEW

File: tests/test_ledger.ml [hunk 1]
```diff
@@ -0,0 +1,186 @@
+open Setup
+open Helpers
+open Crypto
+open Core
+open Ledger
+
+let () =
```
Comment (lykimq) [inline tests/test_ledger.ml:7]: It is a very big function. 
Could it be possible to split them into smaller functions and then each test dedicated for the kind of test that you want to test, in the end, call those test functions?

File: tests/test_tezos_interop.ml [hunk 2]
```diff
@@ -0,0 +1,605 @@
...
+            ()))
+let () =
+  describe "key_hash" (fun { test; _ } ->
+      let open Key_hash in
+      let tz1 = of_key genesis_wallet in
+      let tz2 = of_key TZ2_ex.pk in
+      let tz3 = of_key TZ3_ex.pk in
+      test "to_string" (fun { expect; _ } ->
+          (expect.string (to_string tz1)).toEqual
+            "tz1LzCSmZHG3jDvqxA8SG8WqbrJ9wz5eUCLC";
```
Comment (lykimq) [inline tests/test_tezos_interop.ml:103]: Do you think defining the address name for this is better? for instance (you can have a better name):
`let add_1 = "tz1LzCSmZHG3jDvqxA8SG8WqbrJ9wz5eUCLC"` 
Then when we write the test below it is easier to read the code and know what we are comparing.

**Comment by lykimq** [comment]
General review note: The test functions are very big, I am in favor of splitting them into smaller functions, it is better while reading the code and debugging in case of failure.

**Review by d4hines — APPROVED** [review]
LGTM :+1:

**Review by Zett98 — APPROVED** [review]

File: tests/test_tezos_interop.ml (hunk 2 above)
Comment (EduardoRFS) [reply tests/test_tezos_interop.ml:103]: this is not related to this PR

**Pull request merged**